examples/aldp.py

```diff
@@ -437,11 +486,13 @@ def ais_target_log_prob(x):
             x, log_w, log_q_old, indices = x.to(device), log_w.to(device), \
                                            log_q_old.to(device), indices.to(device)
             log_q_x = model.flow.log_prob(x)
-            # adjustment to account for change to theta since sample was last added/adjusted
-            w_adjust = torch.exp(log_q_old - log_q_x).detach()
-            w_adjust = torch.clip(w_adjust, max=rb_config['max_adjust_w_clip'])
-            # manually calculate the new form of the loss
+            # Adjustment to account for change to theta since sample was last added/adjusted
+            log_w_adjust = log_q_old - log_q_x.detach()
+            w_adjust = torch.clip(torch.exp(log_w_adjust), max=rb_config['max_adjust_w_clip'])
+            # Manually calculate the new form of the loss
             loss = - torch.mean(w_adjust * log_q_x)
+            # Adjust buffer samples
+            buffer.adjust(log_w_adjust, log_q_x.detach(), indices)
 
     else:
```

Comment:
On line 509 I found it useful to check that the gradient norm is not `NaN` before doing an optimizer step (this may not be necessary for aldp problem though)